pkg/install/v1alpha1/localpv_sc.go

```diff
@@ -33,7 +33,7 @@ metadata:
       - name: StorageType
         value: "hostpath"
       - name: BasePath
-        value: "/var/openebs/local"
+        value: {{env "OPENEBS_IO_LOCALPV_HOSTPATH_DIR" | default "/var/openebs/local"}} 
```

Comment:
let us put the sample yaml snippet i.e. maya api server deployment that makes use of this env & value

Reply:
https://github.com/openebs/openebs/pull/2731